src/ethereum_test_specs/base_static.py

```diff
@@ -0,0 +1,229 @@
+            pre: Alloc,
+            n: int,
+            m: int,
+        ):
+            \"\"\"Generate a test from a static state filler.\"\"\"
+            assert n == 1
+            assert m in [1, 2]
+            env = Environment(**self.env.model_dump())
+            sender = pre.fund_eoa()
+            tx = Transaction(
+                ty=0x0,
+                nonce=0,
+                to=Address(0x1000),
+                gas_limit=500000,
+                protected=False if fork in [Frontier, Homestead] else True,
+                data="",
+                sender=sender,
+            )
+            state_test(env=env, pre=pre, post={}, tx=tx)
+
+        return test_state_filler
+        ```
+
+        To aid the generation of the test, the function can be defined and then the decorator be
+        applied after defining the function:
+
+        ```
+        def test_state_filler(
+            state_test: StateTestFiller,
+            fork: Fork,
+            pre: Alloc,
+            n: int,
+            m: int,
+        ):
+            ...
+        test_state_filler = pytest.mark.parametrize("n", [1])(test_state_filler)
+        test_state_filler = pytest.mark.parametrize("m", [1, 2])(test_state_filler)
+        if self.valid_from:
+            test_state_filler = pytest.mark.valid_from(self.valid_from)(test_state_filler)
+        if self.valid_until:
+            test_state_filler = pytest.mark.valid_until(self.valid_until)(test_state_filler)
+        return test_state_filler
+        ```
+
+        The function can contain the following parameters on top of the spec type parameter
+        (`state_test` in the example above):
+        - `fork`: The fork for which the test is currently being filled.
+        - `pre`: The pre-state of the test.
+
+        """
+        raise NotImplementedError
+
+    @staticmethod
+    def remove_comments(data: Dict) -> Dict:
+        """Remove comments from a dictionary."""
+        result = {}
+        for k, v in data.items():
+            if isinstance(k, str) and k.startswith("//"):
+                continue
+            if isinstance(v, dict):
+                v = BaseStaticTest.remove_comments(v)
+            elif isinstance(v, list):
+                v = [BaseStaticTest.remove_comments(i) if isinstance(i, dict) else i for i in v]
+            result[k] = v
+        return result
+
+    @model_validator(mode="before")
+    @classmethod
+    def remove_comments_from_model(cls, data: Any) -> Any:
+        """Remove comments from the static file loaded, if any."""
+        if isinstance(data, dict):
+            return BaseStaticTest.remove_comments(data)
+        return data
+
+
+ALL_FORKS = get_forks()
+
+
+def fork_by_name(fork_name: str) -> Fork:
+    """Get a fork by name."""
+    for fork in ALL_FORKS:
+        if fork.name() == fork_name:
+            return fork
+
+    raise Exception(f'Fork "{fork_name}" could not be identified.')
+
+
+class ForkRangeDescriptor(BaseModel):
```

Comment:
I'd actually suggest moving this to `src/ethereum_test_forks/helpers.py` too :slightly_smiling_face: 

Although I'm struggling to find other use cases tbh. We don't need it in the cli as we have `--from` and `--until`.

But at least we'd be able to find it more easily if we did need it again in the future.

Reply:
Also done in latest commit.